.travis.yml

```diff
@@ -188,6 +188,16 @@ install:
           SYMFONY_VERSION=$(cat composer.json | grep '^ *"dev-master". *"[1-9]' | grep -o '[0-9.]*')
       fi
 
+    - |
+      # Install symfony/flex
+      if [[ $deps = low ]]; then
+          export SYMFONY_REQUIRE='>=2.3'
```

Comment:
should be bumped to `>=2.8` on 3.4 and `>=3.4` on 4.1

Reply:
that looks wrong to me. Forcing `>=3.4` for deps=low on 4.1 might hide the fact that the actual low deps can be older than that based on the requirements for instance..

Reply:
You're right, that happens on deps=low for 4.1.
Let's bump to `>=3.4` when 2.8 is EOLed then, would that be OK for you?

Reply:
even then, it might hide the fact that our low dep is wrong

Reply:
Fine, let's keep it to 2.3 for now...